package/yast2-devtools.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-devtools
-Version:        3.1.46
+Version:        3.1.47
```

Comment:
We should not raise version to test it in YaST:Head first. And if it does not break something *then* submit to Factory.